### ISSUE TITLE
[BUG] System.UnauthorizedAccessException: Access to the path 'RobloxPlayerBeta.exe' is denied.

### ISSUE THREAD

**Issue by Arthur7todfuA7** [bloxstraplabs/bloxstrap#3620]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.
- [X] I have thoroughly looked through the available Wiki articles and could not find a solution to my problem.
- [X] I am using the latest version of Bloxstrap.
- [X] I did not answer truthfully to all the above checkboxes.

### Bloxstrap Version

v2.8.1

### What problem did you encounter?

System.UnauthorizedAccessException: Access to the path 'RobloxPlayerBeta.exe' is denied.

### Bloxstrap Log

```text
2024-11-01T19:27:30Z [App::OnStartup] Starting Bloxstrap v2.8.1
2024-11-01T19:27:30Z [App::OnStartup] Compiled Tuesday, 29 October 2024 at 7:35:23 PM from commit 2be2f4d0d275e60eabf2f63ecbbe4e38b2aa0773 (tag/v2.8.1)
2024-11-01T19:27:30Z [App::OnStartup] Loaded from %UserProfile%\AppData\Local\Bloxstrap\Bloxstrap.exe
2024-11-01T19:27:30Z [App::OnStartup] Temp path is %UserProfile%\AppData\Local\Temp\Bloxstrap
2024-11-01T19:27:30Z [App::OnStartup] WindowsStartMenu path is %UserProfile%\AppData\Roaming\Microsoft\Windows\Start Menu\Programs
2024-11-01T19:27:30Z [LaunchSettings] Identifier 'player' is active
2024-11-01T19:27:30Z [Logger::Initialize] Initializing at %UserProfile%\AppData\Local\Bloxstrap\Logs\Bloxstrap_20241101T192730Z.log
2024-11-01T19:27:30Z [Logger::Initialize] Finished initializing!
2024-11-01T19:27:30Z [JsonManager<Settings>::Load] Loading from %UserProfile%\AppData\Local\Bloxstrap\Settings.json...
2024-11-01T19:27:30Z [JsonManager<Settings>::Load] Loaded successfully!
2024-11-01T19:27:30Z [JsonManager<State>::Load] Loading from %UserProfile%\AppData\Local\Bloxstrap\State.json...
2024-11-01T19:27:30Z [JsonManager<State>::Load] Loaded successfully!
2024-11-01T19:27:30Z [FastFlagManager::Load] Loading from %UserProfile%\AppData\Local\Bloxstrap\Modifications\ClientSettings\ClientAppSettings.json...
2024-11-01T19:27:30Z [FastFlagManager::Load] Loaded successfully!
2024-11-01T19:27:30Z [LaunchHandler::LaunchRoblox] Initializing bootstrapper
2024-11-01T19:27:30Z [LaunchHandler::LaunchRoblox] Initializing bootstrapper dialog
2024-11-01T19:27:30Z [Bootstrapper::Run] Running bootstrapper
2024-11-01T19:27:30Z [Bootstrapper::SetStatus] Conectando-se ao Roblox...
2024-11-01T19:27:30Z [Deployment::TestConnection<https://setup.rbxcdn.com>] Connecting...
2024-11-01T19:27:30Z [HttpClientLoggingHandler::ProcessRequest] GET https://setup.rbxcdn.com/versionStudio
2024-11-01T19:27:30Z [Deployment::InitializeConnectivity] Testing connectivity...
2024-11-01T19:27:30Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://setup.rbxcdn.com/versionStudio
2024-11-01T19:27:30Z [Deployment::InitializeConnectivity] Got https://setup.rbxcdn.com as the optimal base URL
2024-11-01T19:27:30Z [Bootstrapper::Run] Connectivity check finished
2024-11-01T19:27:30Z [Bootstrapper::CheckForUpdates] Checking for updates...
2024-11-01T19:27:30Z [HttpClientLoggingHandler::ProcessRequest] GET https://api.github.com/repos/bloxstraplabs/bloxstrap/releases/latest
2024-11-01T19:27:30Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://api.github.com/repos/bloxstraplabs/bloxstrap/releases/latest
2024-11-01T19:27:30Z [Bootstrapper::CheckForUpdates] No updates found
2024-11-01T19:27:30Z [Bootstrapper::GetLatestVersionInfo] Got channel as production
2024-11-01T19:27:30Z [Deployment::GetInfo] Getting deploy info for channel production
2024-11-01T19:27:30Z [HttpClientLoggingHandler::ProcessRequest] GET https://clientsettingscdn.roblox.com/v2/client-version/WindowsPlayer
2024-11-01T19:27:31Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://clientsettingscdn.roblox.com/v2/client-version/WindowsPlayer
2024-11-01T19:27:31Z [RegistryKeyEx::SetValueSafe] Writing '' to HKEY_CURRENT_USER\SOFTWARE\ROBLOX Corporation\Environments\RobloxPlayer\Channel\www.roblox.com
2024-11-01T19:27:31Z [HttpClientLoggingHandler::ProcessRequest] GET https://setup.rbxcdn.com/version-b7eebc919e96477a-rbxPkgManifest.txt
2024-11-01T19:27:31Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://setup.rbxcdn.com/version-b7eebc919e96477a-rbxPkgManifest.txt
2024-11-01T19:27:31Z [Bootstrapper::SetStatus] Atualizando {product}...
2024-11-01T19:27:31Z [LaunchHandler::LaunchRoblox] Bootstrapper task has finished
2024-11-01T19:27:31Z [LaunchHandler::LaunchRoblox] An exception occurred when running the bootstrapper
2024-11-01T19:27:31Z [App::FinalizeExceptionHandling] (0x80070005) System.UnauthorizedAccessException: Access to the path 'RobloxPlayerBeta.exe' is denied.
   at System.IO.FileSystem.RemoveDirectoryRecursive(String fullPath, WIN32_FIND_DATA& findData, Boolean topLevel)
   at System.IO.FileSystem.RemoveDirectory(String fullPath, Boolean recursive)
   at Bloxstrap.Bootstrapper.UpgradeRoblox()
   at Bloxstrap.Bootstrapper.Run()
   at Bloxstrap.Bootstrapper.Run()
2024-11-01T19:27:31Z [HttpClientLoggingHandler::ProcessRequest] POST https://bloxstraplabs.com/metrics/post-exception
```

**Comment by Arthur7todfuA7** [comment]
System.UnauthorizedAccessException: Access to the path 'RobloxPlayerBeta.exe' is denied.

**Comment by Churrs-ai** [comment]
Hi Does Anyone know why my bloxstrap says this?: System aggregatedexeption one or more errors ocurred the given key content-platform-dictionaries.zip was not present in the dictionairy

**Comment by bluepilledgreat** [comment]
I'm assuming this has been fixed with v2.8.2. If not, please open a new issue.
